### ISSUE TITLE
Derive macro panics if type has doc comment.

### ISSUE THREAD

**Issue by zakarumych** [interact-rs/interact#11]
Error message:
```
 #[cfg_attr(feature = "interact", derive(Interact))]
  |                                         ^^^^^^^^
  |
  = help: message: extend () in `interact` data type attribute `Literal { lit: Str_( Memory utilization stats.), suffix: None, span: Span { lo: BytePos(0), hi: BytePos(0), ctxt: #0 } }`
```

**Comment by da-x** [comment]
Was unable to reproduce the issue. Perhaps can you provide a branch on one of your projects on which this breaks?

**Comment by da-x** [comment]
Never mind - found the issue and fixed in v0.3.3.

**Comment by zakarumych** [comment]
Thanks!
